homework3/task04/tests/test_is_armstrong.py

```diff
@@ -0,0 +1,12 @@
+import pytest
+from project_is_armstrong.task4_armstrong import is_armstrong
+
+
+@pytest.mark.parametrize("number", [153])
+def test_is_armstrong_true(number: int):
+    assert is_armstrong(number) is True, "Is Armstrong number"
+
+
+@pytest.mark.parametrize("number", [10])
```

Comment:
You use a test generator to generate a single test.  I think this is way too complicated.

Reply:
Yes, I fully agree. I dont know why I did in such a strange way.